scripts/zones/AlTaieu/mobs/Jailer_of_Love.lua

```diff
@@ -20,76 +23,287 @@ local minionGroup =
     [7] = 25, -- Qnhpemde
 }
 
+local spellLists =
+{
+    [xi.element.FIRE]    = 508, -- Fire List
+    [xi.element.ICE]     = 509, -- Ice List
+    [xi.element.WIND]    = 510, -- Wind List
+    [xi.element.EARTH]   = 511, -- Earth List
+    [xi.element.THUNDER] = 512, -- Thunder List
+    [xi.element.WATER]   = 513, -- Water List
+    [xi.element.LIGHT]   = 514, -- Light List
+    [xi.element.DARK]    = 515, -- Dark List
+}
+
+-- Animations, action IDs and elemental absorb mods are directly mapped to eachother per retail caps
+-- in the 'standard' order, Fire Ice Wind Earth Thunder Water Holy Dark
+local eleAbsorbActionID   = { 603, 604, 624, 404, 625, 626, 627, 307 }
+local eleAbsorbAnimations = { 432, 433, 434, 435, 436, 437, 438, 439 }
+
+local astralFlowPets = function()
+    for i = ID.mob.JAILER_OF_LOVE + 1, ID.mob.JAILER_OF_LOVE + 27 do
+        local pet = GetMobByID(i)
+        if pet and pet:isAlive() then
+            pet:timer(1500, function(petArg)
+                if petArg:getFamily() == 269 then -- xzomit
+                    petArg:useMobAbility(786) -- lateral slash
```

Comment:
We have enums for these now, can you at these there please?

Reply:
done (this made me look into the fact that Xzomits were told to use a tonberry move. it looks like this section is guessed -- as far as I know, nobody has gotten a capture with  JoL using Astral Flow with an xzomit up)